### PR TITLE
Bring back support for pbjs generated types

### LINKED ISSUE #692: pbjs generated types compatibility.
Hello and excuse me if the answer is obvious as I haven't looked heavily into the inner workings of pbsjs/ts-proto but what is the reason for the switch from `.fromJSON` to `.fromPartial` ?

The reason I'm asking is because generated types from pbjs only output the former function while ones from ts-proto output both functions. Thus keeping `.fromJSON` seems like would aid backwards compatibility.

As it stands we need to update our build pipeline to use ts-proto instead to generate types (which isn't as easy as it sounds) or cosmjs complains with a:
`TypeError: type.fromPartial is not a function`

### PR DESCRIPTION
**Pull request by webmaster128**
Closes #692

### CODE REVIEW

File: packages/proto-signing/src/registry.ts [hunk 1]
```diff
@@ -56,6 +81,22 @@ export class Registry {
     this.types.set(typeUrl, type);
   }
 
+  /**
+   * Looks up a type that was proviously added to the registry.
```
Comment (willclarktech) [inline packages/proto-signing/src/registry.ts:85]: Typo: "previously"

**Review by willclarktech — APPROVED** [review]
Very nice

File: packages/proto-signing/src/registry.ts (hunk 1 above)
Comment (webmaster128) [reply packages/proto-signing/src/registry.ts]: Thanks!

**Comment by fadeev** [comment]
Thanks, @webmaster128 for working on this! 🙏We'll be using protobujs initially, and most likely eventually transition to ts-proto.

**Comment by webmaster128** [comment]
@fadeev I'm happy to support both for the sake of modularaliry and avoiding vendor lock-in.

**Pull request merged**